kmir/src/kmir/kdist/mir-semantics/kmir.md

```diff
@@ -92,6 +92,7 @@ module KMIR-CONFIGURATION
                   // heap
                   <memory> .Map </memory> // FIXME unclear how to model
                   // FIXME where do we put the "GlobalAllocs"? in the heap, presumably?
+                  <start-symbol> $STARTSYM:String </start-symbol>
```

Comment:
Can we maybe just make this cell have sort `Symbol` and insert `symbol("$STARTSYM")`? (instead of adding `symbol(_)` in the code below)